Algorithms/Search/exponential_search.cs

```diff
@@ -0,0 +1,74 @@
+// C# program to find an element x in a 
+// sorted array using Exponential search. 
+using System; 
+class GFG { 
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6263297)